.gitlab-ci.yml

```diff
@@ -271,6 +271,7 @@ py37_main:
     - make test tests=src/toil/test/src
     - make test tests=src/toil/test/utils
     - make test tests=src/toil/test/lib/test_ec2.py
+    - make test tests=src/toil/test/lib/aws
```

Comment:
The new test is already be run as part of the `quick_test_offline `

https://ucsc-ci.com/databiosphere/toil/-/jobs/90265/raw
```
src/toil/test/lib/aws/test_s3.py::S3Test::test_create_bucket 
-------------------------------- live log call ---------------------------------
INFO     toil.test:__init__.py:91 Setting up toil.test.lib.aws.test_s3.S3Test.test_create_bucket ...
INFO     toil.test:__init__.py:96 Tore down toil.test.lib.aws.test_s3.S3Test.test_create_bucket
PASSED 
```

Reply:
But isn't it an online test? Shouldn't it be pulled out of there? `make test_offline` isn't supposed to need Internet access according to the Makefile.

Reply:
I agree, it is an online test. Lots of the `needs_aws`  tests run in the `quick_test_offline`, including the entire `AWSJobStoreTest` suite!

Can we merge this and fix the test distribution in a separate PR?

Reply:
I opened https://github.com/DataBiosphere/toil/issues/3717

Reply:
We might have already had an issue for that; I guess we need a new decorator or something. We probably don't need to fix it here.